### ISSUE TITLE
Remove "Quote" button from message menu

### ISSUE THREAD

**Issue by iprok** [RocketChat/Rocket.Chat#10329]
Hello! 

Seems that "Answer" and "Quote" button do the same, but "Quote" doesn't show preview of quotted message. May be it will be good to remove "Quote" button away?

**Comment by sampaiodiego** [comment]
they have different purposes.. while "reply" button will trigger a notification to the sender of replied message, "quote" is used only to add context.

what we should do is to show the preview for quotes as well, but not remove it.

**Comment by iprok** [comment]
Ok, but you also should edit the documentation. Cause here https://rocket.chat/docs/user-guides/messaging/ we see

**Comment by sampaiodiego** [comment]
right.. I've opened an issue there https://github.com/RocketChat/docs/issues/665

**Comment by geekgonecrazy** [comment]
@karlprieb thoughts?  I would venture to agree that a quote would also show the beautiful new pop up.  Seems weird when quoting and it doesn't show the popup

**Comment by karlprieb** [comment]
@geekgonecrazy I agree :)
